notebook/static/notebook/js/mathjaxutils.js

```diff
@@ -200,7 +205,9 @@ define([
     //
     var replace_math = function (text, math) {
         text = text.replace(/@@(\d+)@@/g, function (match, n) {
-            return math[n];
+            return math[n]
+            	.replace("\\\\\(", "\\\(")
```

Comment:
I believe (following this pattern) this should also cover `.replace("\\\\\[", "\\\[")` and `.replace("\\\\\]", "\\\]")`.

However, I have a code smell around this pattern of sequentially applied string replacement…  

Shouldn't it be sufficient to exclusively looking for these as beginning and ending elements of the string passed into the second function call (since there shouldn't be more than instance of the each to every call to the anonymous function with the `match, n` as an signature). Are there cases where this wouldn't be the prefix and the suffix that are passed into the function that provides the the second part of the `text` return value? 

I should mention, that I would prefer – given that no `this=that` referencing needs to occur inside these anonymous functions –  that the second anonymous function were separately, defined with a proper name. Then the second part of the assignment to the `text` variable would be directly call that function. This would be a lot easier to read.